packages/app-store/webex/lib/VideoApiAdapter.ts

```diff
@@ -0,0 +1,423 @@
+import { z } from "zod";
+
+import dayjs from "@calcom/dayjs";
+import prisma from "@calcom/prisma";
+import type { Credential } from "@calcom/prisma/client";
+import type { CalendarEvent } from "@calcom/types/Calendar";
+import type { CredentialPayload } from "@calcom/types/Credential";
+import type { PartialReference } from "@calcom/types/EventManager";
+import type { VideoApiAdapter, VideoCallData } from "@calcom/types/VideoApiAdapter";
+
+import { getWebexAppKeys } from "./getWebexAppKeys";
+
+/** @link https://developer.webex.com/docs/meetings **/
+const webexEventResultSchema = z.object({
+  id: z.string(),
+  webLink: z.string(),
+  siteUrl: z.string(),
+  password: z.string().optional().default(""),
+});
+
+export type WebexEventResult = z.infer<typeof webexEventResultSchema>;
+
+/** @link https://developer.webex.com/docs/api/v1/meetings/create-a-meeting */
+export const webexMeetingSchema = z.object({
```

Comment:
We just need `start` and `end`. So, we can remove all other props. That should reduce this file's size and simplify it.